ui/app/rif/pages/domainsPage.js.test

```diff
@@ -0,0 +1,114 @@
+import React, { Component } from 'react';
```

Comment:
What is the idea of this js file and why is exactly the same as the `domainsPage.js`?

Reply:
It was just to not loose code that i need... It's deleted now.